doc-source/authorization/authorization_sequence.puml

```diff
@@ -0,0 +1,62 @@
+@startuml
+actor Client
+participant "Servlet" as Servlet
```

Comment:
Hmmm good point. Within the context of the diagram, everything is in the same process. I should rename "Client" to "ClientRequest" since it is Tomcat/Jetty who is providing the input that we care about.

Reply:
Well I guess the Database is external.

Reply:
added a footer to describe